.github/workflows/standard_checks.yml

```
@@ -45,9 +45,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runner: [ ubuntu-24.04, ubuntu-24.04-arm,]
+        runner: [ ubuntu-24.04, ubuntu-24.04-arm,
+                  macos-26, macos-15, macos-14, macos-13 ]
```

```suggestion
        runner:
          - ubuntu-24.04
          - ubuntu-24.04-arm
          - macos-26
          - macos-15
          - macos-14
          - macos-13
```

also, do we really need to support so many versions of macos? I'd say just do 26 and 15 unless you know of someone that is using one of the older ones

---

dropped 14, but kept 15+26 since being different arm64 arch's and 13 being an x86 mac runner